src/template/mapping.json

```diff
@@ -207,6 +207,7 @@
   "diff-removed": 60127,
   "diff-renamed": 60128,
   "diff": 60129,
+  "diff-sidebyside": 60129,
```

Comment:
Do we really need this?

Reply:
I want to change the diff icon eventually, and move current users of `diff` to `diff-sidebyside`.